crates/system_ui/src/mic.rs

```diff
@@ -29,6 +28,8 @@ impl Plugin for MicUiPlugin {
         app.add_systems(Startup, setup.in_set(SetupSets::Main));
         app.add_systems(Update, update_mic_ui);
 
+        app.init_resource::<MicState>();
```

Comment:
let's move this to the comms plugin please, eventually this system ui module will probably get removed entirely